modules/nw-sriov-configuring-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="configuring-sr-iov-devices_{context}"]
+= Configuring the operator
+
+The SR-IOV Network Operator adds the `SriovOperatorConfig.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
+You can configure the SR-IOV software compoenents by modifying the default 
+SriovOperatorConfig Custom Resource (CR).
+
+[NOTE]
+=====
+There is one Custom Resource of SriovOperatorConfig being created when the
+operator is up. User shall change the operator config by modify this pre-created
+Custom Resource.
+=====
+
+== Disabling Network Resource Injector
+
+Network Resources Injector is a Kubernetes Dynamic Admission Controller 
+application that provides functionality of patching pod specifications with 
+requests and limits of custom network resources managed by SRIOV network device
+plugin
+
+By default the Network Resource Injector will be enabled by the operator. You
+can follow this procedure to disable it.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* You must have installed the SR-IOV Operator.
+
+.Procedure
+. Update the default SriovOperatorConfig Custom Resource
++
+[source,bash]
+----
+oc patch sriovoperatorconfig default --patch '{ "spec": { "enableInjector": false}}' --type=merge
+----
+
+== Disabling SRIOV Operator Admission Control Webhook
+The SRIOV Operator Admission Control Webhook a Kubernetes Dynamic 
+Admission Controller application that:
+
+* Validating the SriovNetworkNodePolicy Custom Resource when it being created or
+updated.
+* Mutating the SriovNetworkNodePolicy Custom Resource by setting the default 
+value to `priority` and `deviceType` when it being created or updated.
+
+By default the SRIOV Operator Admission Control Webhook will be enabled by the
+operator. You can follow this procedure to disable it.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* You must have installed the SR-IOV Operator.
+
+.Procedure
+. Update the default SriovOperatorConfig Custom Resource
++
+[source,bash]
+----
+oc patch sriovoperatorconfig default --patch '{ "spec": { "enableOperatorWebhook": false}}' --type=merge
+----
+
+== Configuring Custom NodeSelector for SRIOV Network Config Daemon
+
+The SRIOV Network Config Daemon is in charge of discover and config the SRIOV
```

Comment:
Perhaps:

The SRIOV Network Config Daemon is in charge of discovering and configuring the SRIOV